DESCRIPTION

```diff
@@ -130,3 +130,7 @@ Encoding: UTF-8
 Language: en-US
 RoxygenNote: 7.1.1
 VignetteBuilder: knitr
+Remotes:
```

Comment:
I don't think there is an official "Remotes" tag in R / CRAN, or? So I would rather remove this